project/src/main/java/com/max/grpc/projects/echo/client/EchoClient.java

```diff
@@ -0,0 +1,45 @@
+
+package com.max.grpc.projects.echo.client;
+
+import com.max.grpc.projects.echo.protos.EchoEntity;
+import com.max.grpc.projects.echo.protos.EchoServiceGrpc;
+
+import io.grpc.ManagedChannel;
+import io.grpc.ManagedChannelBuilder;
+import io.grpc.StatusRuntimeException;
+
+import java.util.concurrent.TimeUnit;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+public class EchoClient {
+    private static final Logger logger = Logger.getLogger(EchoClient.class.getName());
+
+    private final ManagedChannel channel;

```

Comment:
May be this variable can have more descriptive name?

Reply:
Sure, renamed